### PR TITLE
Configuring PostgresSql As DbProvider

### PR DESCRIPTION
**Pull request by fsdevrs**
Configuring PostgresSql As DbProvider

### CODE REVIEW

**Comment by fsdevrs** [comment]
Configuring PostgresSql As DbProvider #2 https://github.com/rovinsharmagit/EmployeeManagementSystem/pull/2#issue-2218069179

File: Server/Program.cs [hunk 1]
```diff
@@ -21,10 +21,18 @@
...
 
 //Configure App Db Context
+//builder.Services.AddDbContext<EmpDbContext>(options =>
+//{
+    //options.UseSqlServer(builder.Configuration.GetConnectionString("DevConnections") ?? throw new InvalidOperationException(
+    //  "Sorry , Connection not found"));
+//})
+
+//Configure Connection strings for Postgresql
 builder.Services.AddDbContext<EmpDbContext>(options =>
```
Comment (rovinsharmagit) [inline Server/Program.cs:31]: Comment Out this code section everytime you push your code

**Review by rovinsharmagit — APPROVED** [review]
https://github.com/rovinsharmagit/EmployeeManagementSystem/pull/2#issue-2218069179 PR:Approved & Closed

**Comment by rovinsharmagit** [comment]
https://github.com/rovinsharmagit/EmployeeManagementSystem/pull/2#issue-2218069179

**Pull request merged**